test/orange/Orange.test.cc

```diff
@@ -0,0 +1,167 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2021 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file Orange.test.cc
+//---------------------------------------------------------------------------//
+#include "orange/OrangeParams.hh"
+#include "orange/OrangeTrackView.hh"
+
+#include "base/Constants.hh"
+
+#include "celeritas_test.hh"
+#include "OrangeGeoTestBase.hh"
+// #include "Orange.test.hh"
+
+using namespace celeritas;
+using namespace celeritas_test;
+
+using celeritas::constants::sqrt_two;
+
+//---------------------------------------------------------------------------//
+// TEST HARNESS
+//---------------------------------------------------------------------------//
+
+class OrangeTest : public OrangeGeoTestBase
+{
+  protected:
+    using Initializer_t = GeoTrackInitializer;
+
+    void SetUp() override {}
+
+    //! Create a host track view
+    OrangeTrackView make_track_view()
+    {
+        if (!host_state_)
+        {
+            host_state_ = HostStateStore(this->params(), 1);
+        }
+
+        return OrangeTrackView(
+            this->params().host_ref(), host_state_.ref(), ThreadId{0});
+    }
+
+  private:
+    using HostStateStore
+        = CollectionStateStore<OrangeStateData, MemSpace::host>;
+    HostStateStore host_state_;
+};
+
+class OneVolumeTest : public OrangeTest
+{
+    void SetUp() override
+    {
+        OneVolInput geo_inp;
+        this->build_geometry(geo_inp);
+    }
+};
+
+class TwoVolumeTest : public OrangeTest
+{
+    void SetUp() override
+    {
+        TwoVolInput geo_inp;
+        geo_inp.radius = 1.5;
+        this->build_geometry(geo_inp);
+    }
+};
+
+#define FiveVolumesTest TEST_IF_CELERITAS_JSON(FiveVolumesTest)
+class FiveVolumesTest : public OrangeTest
+{
+    void SetUp() override { this->build_geometry("five-volumes.org.json"); }
+};
+
+//---------------------------------------------------------------------------//
+// TESTS
+//---------------------------------------------------------------------------//
```

Comment:
I don't see anything here that covers the `find_surface` or `bbox` methods. Want to add some quick sanity checks for those?